### PR TITLE
Removing the deprecated micro kernel module 

### PR DESCRIPTION
**Pull request by johanandren**
Refs #22205

### CODE REVIEW

File: akka-kernel/src/main/scala/akka/kernel/Main.scala [hunk 1]
```diff
@@ -1,189 +0,0 @@
...
-  NNNNN$    DNNNNN    NNNNN  $NNNNN~     $NNNN+  NNNNNN      NNNNN,   :NNNNN+
-   ?DN~      NNNNN    NNNNN MNNNNN       $NNNN+:NNNNN7        $ND      =NNNNN
-            DNNNNN    NNNNNDNNNN$        $NNNNDNNNNN                  :DNNNNN
-     ZNDNNNNNNNNND    NNNNNNNNNND,       $NNNNNNNNNNN           DNDNNNNNNNNNN
-   NNNNNNNDDINNNNN    NNNNNNNNNNND       $NNNNNNNNNNND       ONNNNNNND8+NNNNN
- :NNNND      NNNNN    NNNNNN  DNNNN,     $NNNNNO 7NNNND     NNNNNO     :NNNNN
- DNNNN       NNNNN    NNNNN    DNNNN     $NNNN+   8NNNNN    NNNNN      $NNNNN
- DNNNNO     NNNNNN    NNNNN     NNNNN    $NNNN+    NNNNN$   NNNND,    ,NNNNND
-  NNNNNNDDNNNNNNNN    NNNNN     =NNNNN   $NNNN+     DNNNN?  DNNNNNNDNNNNNNNND
-   NNNNNNNNN  NNNN$   NNNNN      8NNNND  $NNNN+      NNNNN=  ,DNNNNNNND NNNNN$
```
Comment (patriknw) [inline akka-kernel/src/main/scala/akka/kernel/Main.scala]: nostalgic

File: akka-docs/rst/java/cluster-usage.rst [hunk 2]
```diff
@@ -666,12 +666,11 @@ Command Line
   in the next major version. Use the :ref:`cluster_http_java` API with `curl <https://curl.haxx.se/>`_
   or similar instead.
 
-The cluster can be managed with the script `bin/akka-cluster` provided in the
-Akka distribution.
+The cluster can be managed with the script ``akka-cluster`` provided in the Akka github repository here: @github@/akka-cluster/jmx-client. Place the script and the ``jmxsh-R5.jar`` library in the same directory.
```
Comment (patriknw) [inline akka-docs/rst/java/cluster-usage.rst]: the script needs to be adjusted, it is using `/bin/jmxsh-R5.jar`

**Review by patriknw — CHANGES_REQUESTED** [review]

File: akka-kernel/src/main/scala/akka/kernel/Main.scala (hunk 1 above)
Comment (ktoso) [inline akka-kernel/src/main/scala/akka/kernel/Main.scala]: :D

File: akka-docs/rst/java/cluster-usage.rst (hunk 2 above)
Comment (johanandren) [inline akka-docs/rst/java/cluster-usage.rst]: `$AKKA_HOME/bin` even.

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by ktoso** [comment]
https://twitter.com/propensive/status/552219523156176897 
jon_pretty_on_twitter___whenever_akka_fails_on_startup__this_is_what_my_eyes_do__aaaaarrrrrgggghhhhh_____cc__viktorklang__jboner__rolandkuhn_http___t_co_m4zpye2yu1_

LGTM.

**Pull request merged**